sem_covid/services/pwdb_base_experiment.py

```diff
@@ -66,6 +67,7 @@ def data_extraction(self, *args, **kwargs):
         raw_pwdb_dataset = self.requests.get(config.PWDB_DATASET_URL, stream=True, timeout=30)
```

Comment:
I just realised that we no longer need to load the RAW dataset. 
The extraction step is in fact already executed by the PWDB ETL DAG and we can just use the Dataset.PWDB.fetch() for accessing the data.

Reply:
so this step can remain empty